architecture_reviews/data_act_pilot/README.md

```diff
@@ -57,7 +54,7 @@ described viewing code as “not real” to be a trap. Their small code base
 quickly grew larger than anticipated and their early technical decisions would
 have long lasting ramifications.
 
-![Diagram matching above description](../../assets/img/data-act-diagram.svg)
+![Diagram matching above description](/architecture_reviews/data_act_pilot/data-act-diagram.png)
 
```

Comment:
PNG's a lossy format -- I figure if we keep the SVGs, they can be edited and exported again in the future.